extensions/converter/src/byte-converter.tsx

```diff
@@ -0,0 +1,141 @@
+          value = value.replace(/[^\d.]/g, ""); // Should only contains numbers
+          const parsedValue = parseFloat(value);
+          if (isNaN(parsedValue)) {
+            console.info(`newValue "${newValue}" is not a valid number`);
+            return;
+          }
+          console.debug(newValue, value, parsedValue);
+          const index = units.indexOf(unit);
+          const bytesValue = convertToBytes(parsedValue, index);
+
+          setState((prevState) => {
+            const newState = { ...prevState };
+            for (let i = 0; i < index; i++) {
+              newState[units[i]] = convert(bytesValue, i);
+            }
+            newState[unit] = parsedValue;
+            for (let i = index + 1; i < units.length; i++) {
+              newState[units[i]] = convert(bytesValue, i);
+            }
+            return newState;
+          });
+        },
+      };
+    }, {} as State & SetFunctions<State>)
+  );
+
+  const textFields = useRef<(Form.TextField | null)[]>(Array.from({ length: units.length }));
+
+  const [focusedTextFieldIndex, setFocusedTextFieldIndex] = useState<number>(0);
+
+  const { autoDetect, priorityDetection } = commonPreferences();
+  const inputItem = getInputItem(autoDetect, priorityDetection);
+  useEffect(() => {
+    async function _fetch() {
+      state["setBytes"](inputItem);
+    }
+
+    _fetch().then();
+  }, [inputItem]);
+
+  const getBestUnitExpression = () => {
+    const values = units.map((unit) => state[unit]);
+    const maxValue = 1000;
+    let bestIndex = 0;
+    for (; bestIndex < units.length - 1; bestIndex++) {
+      const value = values[bestIndex];
+      if (value < maxValue) {
+        break;
+      }
+    }
+    return getExpressionAtIndex(bestIndex);
+  };
+
+  const getFocusedValueExpression = () => {
+    return getExpressionAtIndex(focusedTextFieldIndex);
+  };
+
+  const getExpressionAtIndex = (index: number): string => {
+    const unit = units[index];
+    const value = state[unit];
+    const expression = `${value} ${unit}`;
+    return expression;
+  };
+
+  return (
+    <Form
+      actions={
+        <ActionPanel>
+          <Action.CopyToClipboard
+            title={`Copy ${getBestUnitExpression()}`}
+            content={getBestUnitExpression()}
+            shortcut={{ modifiers: [], key: "enter" }}
+          />
+          <Action.CopyToClipboard title={`Copy ${getFocusedValueExpression()}`} content={getFocusedValueExpression()} />
+          <ActionPanel.Section>
+            {units.map((_, index) => {
+              const keyEquivalent = KeyEquivalentByNumber(index + 1);
+              return (
+                <Action.CopyToClipboard
+                  key={index} // to make key unique to avoid warning
+                  title={`Copy ${getExpressionAtIndex(index)}`}
+                  content={getExpressionAtIndex(index)}
+                  shortcut={keyEquivalent ? { modifiers: ["cmd"], key: keyEquivalent } : undefined}
```

Comment:
cmd+1/2/3/4/5/6/7/8/9/0 are already used by Raycast whenever a list is present, so this shortcut should be changed to something else (e.g. cmd+shift+1)